src/cascadia/TerminalSettingsModel/Theme.cpp

```diff
@@ -119,12 +92,10 @@ til::color ThemeColor::ColorFromBrush(const winrt::WUX::Media::Brush& brush)
     return {};
 }
 
-winrt::WUX::Media::Brush ThemeColor::Evaluate(const winrt::WUX::ResourceDictionary& res,
+winrt::WUX::Media::Brush ThemeColor::Evaluate(const winrt::WUX::ResourceDictionary&,

```

Comment:
```suggestion
winrt::WUX::Media::Brush ThemeColor::Evaluate(
```
why not remove the first param entirely?

Reply:
We may want to leave that for theme colors that reference XAML keys. That was a wacky proposal early in the themes spec that hasn't been implemented yet